src/index.js

```diff
@@ -33,8 +33,8 @@ i18n.init().then(() => {
   ReactDOM.render(
   <Provider store={store}>
     <App />
-  </Provider>, document.getElementById('root'));
-})
```

Comment:
Does the app still run? I needed to have these brackets/braces here or it would give me an error in vscode